.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+/db/migrations/
```

Comment:
The line `/db/migrations/` should be corrected to `db/migrations/` to ensure that the migrations directory is properly ignored by Git.